src/renderer/api/api-manager.ts

```diff
@@ -18,7 +18,7 @@ export class ApiManager {
   }
 
   getApiByKind(kind: string, apiVersion: string) {
-    return Array.from(this.apis.values()).find((api) => api.kind === kind && api.apiVersion === apiVersion);
+    return Array.from(this.apis.values()).find((api) => api.kind === kind && api.apiVersionWithGroup === apiVersion);
```

Comment:
I think we need to backport this fix to 4.0.x series.

Reply:
Hi @jakolehm. Could you please elaborate? Should I create a separate PR fox this fix?

Reply:
Sorry for not being clear.. it was just a reminder for maintainers that we need to backport this fix 🙂 I think maintainers can do it because it needs to target different branch etc.

Reply:
got it, thank you 🙂